rolling/distribution.yaml

```
@@ -4404,7 +4404,8 @@ repositories:
     release:
       tags:
         release: release/rolling/{package}/{version}
-      url: https://github.com/ros2-gbp/rc_dynamics_api-release.git
+      url: git@github.com:ros2-gbp/rc_dynamics_api-release.git
```

```suggestion
      url: https://github.com/ros2-gbp/rc_dynamics_api-release.git
```

The repository url in the distribution.yaml should remain https so it can be fetched without authentication. (Even public repositories require ssh auth for operations)

You can either use a `[url "ssh://git@github.com/"] insteadOf = "https://github.com/"` in your .gitconfig ([docs](https://git-scm.com/docs/git-config#Documentation/git-config.txt-urlltbasegtinsteadOf)) or use the bloom `--override-release-repository-push-url` flag to specify an ssh url.

---

Thanks! Actually meant to use `--override-release-repository-push-url` but instead used `--override-release-repository-url`.